backends/fs/abstract-fs.h

```diff
@@ -124,7 +124,7 @@ class AbstractFSNode {
 	 *
 	 * @note By default, this method returns the value of getName().
 	 */
-	virtual Common::String getDisplayName() const { return getName(); }
+	virtual Common::U32String getDisplayName() const { return getName(); }
```

Comment:
Maybe this could be made into a pure virtual function? The Dreamcast backend seems to be the only platform that doesn't provide an implementation of this.

Reply:
Yes, this is a sound suggestion.